src/o365/spo/commands/site/site-inplacerecordsmanagement-set.ts

```diff
@@ -0,0 +1,144 @@
+import auth from '../../SpoAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import * as request from 'request-promise-native';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import { Auth } from '../../../../Auth';
+import Utils from '../../../../Utils';
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  siteUrl: string;
+  enabled: string;
+}
+
+class SpoSiteInPlaceRecordsManagementSetCommand extends SpoCommand {
+  public get name(): string {
+    return commands.SITE_INPLACERECORDSMANAGEMENT_SET;
+  }
+
+  public get description(): string {
+    return 'Activates or deactivates in-place records management for a site collection';
+  }
+
```

Comment:
We should track the value of the `enabled` property in our telemetry to understand the usage of this command